MTG_Cards/Sources/ViewControllers/MTGDetailViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  MTGDetailViewController.swift
+//  MTG_Cards
+//
+//  Created by Dmitriy Baskakov on 07.09.2022.
+//
+
+import UIKit
+
+class MTGDetailViewController: UIViewController {
+    
+    //MARK: - Properties
+    var card: Displayed?
+    let nameLabel = MTGTitleLabel(fontSize: 30)
+    let setNameLabel = MTGTitleLabel(fontSize: 26)
+    let typeLabel = MTGBodyLabel(fontSize: 24)
+    let manaCostLabel = MTGBodyLabel(fontSize: 24)
+    let powerLabel = MTGBodyLabel(fontSize: 24)
+    let textLabel = MTGBodyLabel(fontSize: 22)
+    let numberLabel = MTGBodyLabel(fontSize: 20)
+    
+    var viewsArray: [UIView] = []
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        set(with: card)
+        configureView()
+    }
+    
+    //MARK: - Methods
+    //Set card info
+    func set(with card: Displayed?) {
+        nameLabel.text = card?.nameLabel
+        setNameLabel.text = card?.setNameLabel
+        typeLabel.text = card?.typeLabel
+        manaCostLabel.text = card?.manaCostLabel
+        powerLabel.text = card?.powerLabel
+        textLabel.text = card?.textLabel
+        numberLabel.text = card?.numberLabel
+    }
+    
+    //Layouts
+    private func configureView() {
+        view.backgroundColor = .systemBackground
+        title = String("Info about: \((card?.nameLabel) ?? "No Name")")
+        
+        view.addSubview(nameLabel)
+        view.addSubview(setNameLabel)
+        view.addSubview(numberLabel)
+        
+        viewsArray = [typeLabel, manaCostLabel, powerLabel, textLabel]
+        
+        for views in viewsArray {
+            view.addSubview(views)
+            
+            NSLayoutConstraint.activate([
+                views.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Metrics.cardInfoLeadingAndTrailing),
+                views.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Metrics.cardInfoLeadingAndTrailing)
+            ])
+        }
```

Comment:
Можно вот так записать тогда viewsArray не потребуется
```suggestion
       [typeLabel, manaCostLabel, powerLabel, textLabel].forEach {
            view.addSubview($0)
            NSLayoutConstraint.activate([
                $0.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Metrics.cardInfoLeadingAndTrailing),
                $0.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Metrics.cardInfoLeadingAndTrailing)
            ])
        }
```